Event/RefreshTokenEvent.php

```diff
@@ -14,7 +14,7 @@
 use Gesdinet\JWTRefreshTokenBundle\Model\RefreshTokenInterface;
 use Symfony\Component\Security\Guard\Token\PostAuthenticationGuardToken;
 
-class RefreshEvent extends Event
+class RefreshTokenEvent extends Event
```

Comment:
It's backward compatibility. You should make a new class and extend it with `RefreshEvent` and trigger a deprecation on the old one (in phpdoc or construct i don't know how it's managed for this project).

Reply:
Okay thanks!